cmd/ir2proxy/main.go

```diff
@@ -53,7 +53,10 @@ func run() int {
 			return 1
 		}
 
-		hp, translationErrors := translator.IngressRouteToHTTPProxy(ir)
+		hp, translationErrors, err := translator.IngressRouteToHTTPProxy(ir)
```

Comment:
nit: they are not errors, they are warnings.

This is ok for now, please fix it in a followup.